app/code/Transoft/Blog/etc/db_schema.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="transoft_blog" resource="sales" engine="innodb" comment="Transoft Blog">
```

Comment:
Why did you specify **sales** connection?

Reply:
fixed